ionic2/app/components/poke-filter-popover/poke-filter-popover.component.ts

```diff
@@ -32,7 +21,7 @@ import { FilterService } from '../../services/filter.service';
   ],
 })
 
-export class FilterPopoverComponent {
+export class PokeFilterPopoverComponent {
```

Comment:
No. It's good practice to prefix the selectors, but Prefixing Component names was a bad idea in the first place.